src/StringRenderer.ts

```diff
@@ -1,16 +1,24 @@
 import type { Renderer, RSTNode } from 'enzyme';
 import type { ReactElement } from 'react';
 import { h, render } from 'preact';
+import renderToString from 'preact-render-to-string';
 
 import type { EventDetails } from './MountRenderer';
+import type { PreactAdapterOptions } from './Adapter';
 
 export default class StringRenderer implements Renderer {
+  constructor(private _options: PreactAdapterOptions) {}
```

Comment:
Mmm, I'm not a huge fan of this shorthand that TS provides for declaring class fields, especially given the differing naming conventions for constructor arguments and field names.